rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java

```diff
@@ -529,8 +533,7 @@ private void restoreNodes(NodeSource nodeSource) {
             totalRecoveredNodes += nodeStateIntEntry.getValue();
         }
         logger.info("Total number of nodes recovered: " + totalRecoveredNodes + ", including eligible nodes: " +
```

Comment:
You should use the constant `END_OF_NODES_RECOVERY` here.

Reply:
Thanks! Fixed